verilog/tools/ls/symbol-table-handler.cc

```diff
@@ -74,33 +77,27 @@ void SymbolTableHandler::LoadProjectFileList(absl::string_view current_dir) {
   LOG(INFO) << __FUNCTION__;
   if (!currproject) return;
   // search for FileList file up the directory hierarchy
-  std::filesystem::path currentdirpath{current_dir.begin(), current_dir.end()};
-  std::filesystem::path projectpath = currentdirpath / filelistname;
   FileList filelist;
-  while (true) {
-    projectpath = currentdirpath / filelistname;
-    // file found
-    LOG(INFO) << "Checking existence of " << projectpath;
-    if (std::filesystem::exists(projectpath)) break;
-    // file not found, reached root directory
-    if (currentdirpath.parent_path() == currentdirpath) {
-      LOG(INFO) << filelistname << " not found";
-      return;
-    }
-    currentdirpath = currentdirpath.parent_path();
+  std::string projectpath;
+  if (verible::file::UpwardFileSearch(current_dir,
```

Comment:
Just a suggestion, feel free to ignore:

You can also use the `ok()` boolean return. So if-statements can read as `if (somethingsomething.ok())`

So here, this could be: 
```c++
if (!verible::file::UpdwardFileSearch(...).ok()) {
}
```

In general, it is probably useful to output the message from the status object in the logmessage. A typical code pattern would be

```c++
if (auto status = verible::file::UpwardFileSearch(...); !status.ok()) {
   LOG(WARNING) << "..." << status;
   return;
}
```